Train_Colab.ipynb

```diff
@@ -343,27 +380,31 @@
         "shuffle_tags = False #@param{type:\"boolean\"}\n",
         "#@markdown * You can turn off the text encoder training (generally not suggested)\n",
         "Disable_text_Encoder= False #@param{type:\"boolean\"}\n",
-        "#@markdown * Skip the nth last layer of CLIP. 0 is default for SD1.x, 2 recommended for SD2.x models.\n",
-        "Clip_skip = 0 #@param {type:\"slider\", min:0, max:4, step:1}\n",
-        "#@markdown * Ratio of training on empty caption. Improves unconditional guidance.\n",
-        "Conditional_DropOut = 0.04 #@param {type:\"slider\", min:0, max:0.3, step:0.01}\n",
-        "#@markdown * Ratio of images randomly to flip horizontally.  Use for small data sets. May negtively affect likeness of subjects with asymmetric features.\n",
-        "Picture_flip = 0.0 #@param {type:\"slider\", min:0, max:0.5, step:0.05}\n",
-        "#@markdown * This can improve contrast in light and dark scenes, Use a ratio between 0-10% for Best results.  0\n",
-        "zero_frequency_noise = 0.05 #@param {type:\"slider\", min:0, max:0.25, step:0.01}\n",
+        "#@markdown * Skip the nth last layer of CLIP.\n",
+        "Clip_skip = 1 #@param {type:\"slider\", min:0, max:4, step:1}\n",
+        "#@markdown * ratio of captions dropped from training data.\n",
+        "Conditional_DropOut = 0.1 #@param {type:\"slider\", min:0, max:0.3, step:0.01}\n",
+        "#@markdown * Ratio of images randomly to flip horizontally.\n",
+        "Picture_flip = 0.05 #@param {type:\"slider\", min:0, max:0.5, step:0.05}\n",
```

Comment:
I think flip should either be zero or 0.50, I recommend leaving it 0.0 as thats the default for the CLI trainer.